data.sql

```diff
@@ -12,5 +12,41 @@ VALUES
 ('Angemon', 'Jun 12, 2005', -45, 'true', 1),
 ('Boarmon', 'Jun 7, 2005', 20.4, 'true', 7),
 ('Blossom', 'Oct 13, 1998', 17, 'true', 3),
-('Ditto', 'May 14, 2022', 22, 'true', 4)
-;
\ No newline at end of file
+('Ditto', 'May 14, 2022', 22, 'true', 4);
+
+
+INSERT INTO owners (full_name, age)
+VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species (name)
+VALUES ('Pokemon'), ('Digimon');
+
+BEGIN;
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon')
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Pokemon')
+WHERE species_id IS NULL;
+COMMIT;
+
+BEGIN;
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name = 'Agumon';
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name = 'Angemon' OR name = 'Boarmon';
+COMMIT;
```

Comment:
- [ ] KIndly add an update statement for the`id` from owners where the `full_name` = `'Bob'`

You have added Bob to the owner's table but you are missing the update statement
![image](https://user-images.githubusercontent.com/81932478/222111891-e262caf9-8e38-4f67-9330-47cfbe6b16da.png)